Library/Homebrew/sbom.rb

```diff
@@ -216,25 +279,28 @@ def full_spdx_runtime_dependencies
       bottle_info = get_bottle_info(dependency["bottle"])
       next unless bottle_info.present?
 
+      # Only set bottle URL if the dependency is the same version as the formula/bottle.
+      bottle_url = bottle_info["url"] if dependency["pkg_version"] == dependency["formula_pkg_version"]
```

Comment:
What is this check for?

Reply:
@SMillerDev You can end up with a formula/keg that was built against an older version of a dependency where we can no longer get a correct URL.

Reply:
Ah... because it's no longer exclusively at build time.